src/main/java/Floda.java

```diff
@@ -0,0 +1,83 @@
+import java.util.Scanner;
+
+public class Floda {
+    public static void main(String[] args) {
```

Comment:
Consider refactoring the code in main for easier readability
eg. displayWelcomeMessage(), getUserInput(), displayExitMessage()